### ISSUE TITLE
PR to remove retired REST API specs is marked BreakingChangeReviewRequired

### ISSUE THREAD

**Issue by mikekistler** [Azure/azure-sdk-tools#9336]
[REST API specs PR 19679](https://github.com/Azure/azure-rest-api-specs-pr/pull/19679) is removing retired versions of REST API definitions from the repo and has been flagged as "BreakingChangeReviewRequired".

We created a label "Versioning-Approved-Retired" ([Issue #30652](https://github.com/Azure/azure-rest-api-specs/issues/30652)) to use on PRs that remove retired API definitions from the repo, on the assumption that these would be flagged as "VersioningReviewRequired".

So either the above PR _should have been flagged_ as "VersioningReviewRequired" or we need to add support for a "BreakingChange-Approved-Retired" label.

cc: @JeffreyRichter

**Comment by JeffreyRichter** [comment]
Yes, I agree. I think it would be better if the tool added the VersioningReviewRequired label since deleting swaggers is NOT creating a new version of anything but changing existing version(s).
If, for some reason, this is too difficult to fix we could live with BreakingChangeReviewRequired and then apply a new BreakingChange-Approved-Retired label but this is not preferred.
